src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -265,7 +266,11 @@ def handle_update_case_command(
         Context(
             elements=[
                 MarkdownText(
-                    text=f"Note: Cases cannot be escalated here. Please use the `{context['config'].slack_command_escalate_case}` slash command."
```

Comment:
This wasn't provided in button clicks from what I could tell, so I switched to using the `SlackConversationConfiguration`. I'm open to other implementations.